ui/de-lib/src/main/java/org/iplantc/de/admin/apps/client/presenter/grid/AdminAppsGridPresenterImpl.java

```diff
@@ -70,6 +74,47 @@ public void onSuccess(AppDoc result) {
         }
     }
 
+    class SetAppAvusCallback implements AsyncCallback<List<Avu>> {
+        App app;
+
+        public SetAppAvusCallback(App app) {
+            this.app = app;
+        }
+
+        @Override
+        public void onFailure(Throwable caught) {
+            ErrorHandler.post(caught);
+        }
+
+        @Override
+        public void onSuccess(List<Avu> result) {
+            app.setBeta(false);
+            listStore.update(app);
+            announcer.schedule(new SuccessAnnouncementConfig(appearance.betaTagRemovedSuccess()));
+        }
+    }
+
+    class AddAppAvusCallback implements AsyncCallback<List<Avu>> {
```

Comment:
Maybe these callbacks could be named something more specific for setting and removing the Beta AVU.
When I first looked at these callbacks, I thought they were generic metadata AVU endpoint callbacks, until I saw them setting the beta flag of the app, which was a little confusing at first.